### PR TITLE
Adding in Ink Mainnet Network Information

### PR DESCRIPTION
**Pull request by alexanderbradshaw**
Add the Ink Mainnet network information so devs can build on top of Ink

### CODE REVIEW

File: pnpm-lock.yaml
Comment (jxom) [inline pnpm-lock.yaml:1]: please revert this change

**Comment by jxom** [comment]
chain also needs to be added to `src/chains/index.ts`

**Comment by jxom** [comment]
please reopen when comments are addressed

**Comment by thINKoor** [comment]
Addressed the changes in the following PR: https://github.com/wevm/viem/pull/3141
